test/e2e/fixtures.go

```diff
@@ -163,6 +163,48 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error) {
 	return data, nil
 }
 
+func setupTestWithFlowAggregator(tb testing.TB) (*TestData, error) {
+	data := &TestData{}
+	if err := data.setupLogDirectoryForTest(tb.Name()); err != nil {
+		tb.Errorf("Error creating logs directory '%s': %v", data.logsDirForTestCase, err)
+		return nil, err
+	}
+	tb.Logf("Creating K8s clientset")
+	if err := data.createClient(); err != nil {
+		return nil, err
+	}
+	tb.Logf("Creating '%s' K8s Namespace", testNamespace)
+	if err := data.createTestNamespace(); err != nil {
+		return nil, err
+	}
+	// Create pod using ipfix collector image
+	if err := data.createPodOnNode("ipfix-collector", masterNodeName(), ipfixCollectorImage, nil, nil, nil, nil, true, nil); err != nil {
+		tb.Fatalf("Error when creating the ipfix collector Pod: %v", err)
+	}
+	ipfixCollectorIP, err := data.podWaitForIPs(defaultTimeout, "ipfix-collector", testNamespace)
+	if err != nil {
+		tb.Fatalf("Error when waiting to get ipfix collector Pod IP: %v", err)
+	}
+	ipStr := ipfixCollectorIP.ipv4.String()
+	tb.Logf("Applying flow aggregator YAML with ipfix collector address: %s", ipStr)
+	if err := data.deployFlowAggregator(ipStr + ":" + ipfixCollectorPort + ":tcp"); err != nil {
+		return data, err
+	}
+	flowAggregatorIP, err := data.getServiceIP("flow-aggregator", flowAggregatorNamespace)
```

Comment:
I see e2e test failures on kind setups in this PR.. both for TestFlowExporter and TestFlowAggregator.
Seems like it needs some debugging.
 
Could you cherry-pick my commit (https://github.com/vmware-tanzu/antrea/pull/1629) and make changes for e2e tests? You will not need to acquire serviceIP of flow aggregator.